### PR TITLE
LPS-162194 | Test Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
[LPS-162194](https://issues.liferay.com/browse/LPS-162194) - Automate ObjectFields#CanCreateActionWithCustomFieldInSystemObjects

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -122,6 +122,62 @@ definition {
...
 	}
 
+	@description = "LPS-159151 - Verify that it's possible to create an Action with a custom field in a System Object"
+	@priority = "4"
+	test CanCreateActionWithCustomFieldInSystemObjects {
+		task ("Add a Custom Object and Custom Field (API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 151573",
+				objectName = "CustomObject151573",
+				pluralLabelName = "Custom Objects 151573");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:132]: Use the ticket number correct to add the Custom Object - [LPS-162194](https://issues.liferay.com/browse/LPS-162194)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 2]
```diff
@@ -122,6 +122,62 @@ definition {
...
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field A",
+				fieldName = "customObjectFieldA",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject151573");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:140]: Also too - [LPS-162194](https://issues.liferay.com/browse/LPS-162194)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 3]
```diff
@@ -122,6 +122,62 @@ definition {
...
+			ObjectPortlet.selectCustomObject(label = "Account");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Custom Action",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "Action shot on add",
+				objectLabel = "Custom Object 151573",
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:165]: Also too - [LPS-162194](https://issues.liferay.com/browse/LPS-162194)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 4]
```diff
@@ -122,6 +122,62 @@ definition {
...
+				thenAction = "Add an Object Entry",
+				whenAction = "On After Add");
+		}
+
+		task ("Add an entry and assert if the action was triggered") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Account Name",
+				accountEntryType = "Business");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject151573");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:175]: Also too - [LPS-162194](https://issues.liferay.com/browse/LPS-162194)

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hi @rafaellajordao,

The testcases were well written. Added some comments to avoid CI errors. Check and fix it. Thanks!

**Comment by rafaellajordao** [comment]
conflict resolved in this https://github.com/liferay-bpm-qa/liferay-portal/pull/279

**Pull request closed without merging**